hotel/models.py

```diff
@@ -1,3 +1,96 @@
 from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
+from django.contrib.auth.models import User,AbstractUser
+from django.utils import timezone
+import uuid
 
-# Create your models here.
+class User(AbstractUser):
+    phoneNumber = PhoneNumberField(unique=True)
```

Comment:
- custom dùng field email để login
- chưa run lại migrate đúng ko
```suggestion
    phoneNumber = models.PhoneNumberField(unique=True)
```